packages/sdks/output/nextjs/README.md

```diff
@@ -1,71 +1,16 @@
-# Builder.io React NextJS SDK (EXPERIMENTAL)
+⚠️: Please read the below carefully.
 
-This is the Builder NextJS SDK, `@builder.io/sdk-react-nextjs`. It is intended to be used _only_ with NextJS's app directory, and has hard dependencies on NextJS-specific functionality that only works in the app directory.
-
-## Note
-
-You can use _any_ Builder React SDK with Next.js. Our [Gen1](../../../react/) and [Gen2](../react/) SDKs both work with Next.js (pages router or app router).
-
-This NextJS SDK, though, is the only one that supports [registering](https://www.builder.io/c/docs/custom-components-setup) RSCs (react server components) as custom components in Builder, and also includes nearly 0 client side JS runtime. Our other SDKs only support registering React client components in Builder, due to how RSC's runtime works.
-
-To allow registering RSCS, this SDK makes compromises on Builder features: most notably, it does not support interactivity within the rendered content. See the [features grid](https://github.com/BuilderIO/builder/tree/main/packages/sdks#features) for more information.
-
-## API Reference
-
-To use the SDK, you need to:
-
-- fetch the builder data using `getContent`: you can see how to use it here https://www.builder.io/c/docs/content-api, and how it differs from the React V1 SDK's `builder.get()` function.
-
-- pass that data to the `RenderContent` component, along with the following properties:
+- you should ONLY use this SDK if you are trying to [register](https://www.builder.io/c/docs/custom-components-setup) your RSCs (react server components) in Builder. That is its only advantage over our standard React SDKs.
+- our [Gen1](../../../react/) and [Gen2](../react/) React SDKs work perfectly well with all versions of Next.js. The only feature they do not support is registration of RSCs.
+- to allow registering RSCs, this SDK must make compromises. Most notably, it does not support interactivity within the rendered content. See the [features grid](https://github.com/BuilderIO/builder/tree/main/packages/sdks#features) for more information.
```

Comment:
this makes it sound like nothing can be interactive, but you can use client component sthat are interactive in hte editor. just doesn't allow interactivity created direclty in builder, which is kind of a rare/power feature anyway 

I think the other thing to call out is the editor will be noticably more laggy, because we have to do round trips to the server for many of the updates